renderer.tsx

```diff
@@ -61,20 +61,30 @@ export default class StarboardExtension extends LensRendererExtension {
 
     clusterPageMenus = [
         {
+            id: "starboard",
+            title: "Starboard",
+            components: {
+                Icon: CertificateIcon
+            }
+        },
+        {
+            parentId: "starboard",
             target: {pageId: "vulnerabilityreports"},
             title: "VulnerabilityReports",
             components: {
                 Icon: CertificateIcon
```

Comment:
@jakolehm  Do we still need to provide icons at sub-menu level? It seems that they do not show up anyway?

![sub-menu-icons](https://user-images.githubusercontent.com/1322923/99801704-c3509a80-2b36-11eb-8f60-dc8383b1a368.png)

Reply:
Icon is still required, we probably will change that to optional before GA.